jobs/JGDAS_GLOBAL_AERO_ANALYSIS_FINALIZE

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+#############################
```

Comment:
@CoryMartin-NOAA 
Please take a look at [JGLOBAL_ATMOS_ANALYSIS](https://github.com/NOAA-EMC/global-workflow/blob/develop/jobs/JGLOBAL_ATMOS_ANALYSIS) for the way the j-job is structured.
We are now following the NCO EE2 recommended structure for all j-jobs.

Reply:
Do you want the order followed exactly in which these things are defined?

Reply:
@aerorahul let me know if the most recent changes address what you were thinking here

Reply:
yep. this does it. Thanks.

I just now realized that we may have multiple `DATA` directories for the same job name and cycle for the different `CDUMP` values.
I think we need to distinguish that in line 9, but cannot be done as `${CDUMP}aeroanl_${cyc}`.
Ops uses `RUN` and we use/set `CDUMP=RUN` internally much later.

Reply:
yeah good point, we will need that solved.